eoscostarica/lifebank
# Feat/lifebank requirements

Resolves #487 

### What does this PR do?
Add requirement field to lifebank profile and info lifebank page

### How should this be tested?
- Make run
- Go to lifebank profile
- Add a new requirement
- Check that is visible in the lifebank info page
